scripts/update-relay-servers-tokens.ts

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable node/no-unsupported-features/es-syntax */
+/* eslint-disable node/no-unpublished-import  */
+/* eslint-disable node/no-missing-import */
+
+import { RelayAdmin, Token, PriceOracleTicker } from './relay/relay-admin.js';
+import { readFileSync } from 'fs';
+
+if (process.argv.length < 6) {
+  console.error(
+    'usage: yarn update-relay-tokens [tokenlist.json] [host] [username] [password]\n' +
```

Comment:
Let's make the command `yarn update-relay-gas-tokens` and the script file name `update-relay-server-gas-tokens.ts`